modules/@apostrophecms/asset/index.js

```diff
@@ -286,15 +373,43 @@ module.exports = {
             return output;
           }
 
-          function merge() {
+          function merge(scene) {
+            const jsModules = `${scene}-module-bundle.js`;
+            const jsNoModules = `${scene}-nomodule-bundle.js`;
+            const css = `${scene}-bundle.css`;
+            fs.writeFileSync(
+              `${bundleDir}/${jsModules}`,
+              Object.entries(self.options.builds).filter(
+                ([ name, options ]) => options.scenes.includes(scene) &&
+                options.outputs.includes('js') &&
+                (!options.condition || options.condition === 'module')
+              ).map(([ name, options ]) => {
+                return `// BUILD: ${name}\n` + fs.readFileSync(`${bundleDir}/${name}-build.js`, 'utf8');
+              }).join('\n')
+            );
+            fs.writeFileSync(
+              `${bundleDir}/${jsNoModules}`,
```

Comment:
The jsNoModules bundle contains code from builds that have "condition: nomodule" (the src-es5 build) and those that don't care. Thus everything IE11 can handle.